packages/mui-joy/src/Breadcrumbs/Breadcrumbs.test.js

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createRenderer, describeConformance } from 'test/utils';
+import { ThemeProvider } from '@mui/joy/styles';
+import Breadcrumbs, { breadcrumbsClasses as classes } from '@mui/joy/Breadcrumbs';
+import { unstable_capitalize as capitalize } from '@mui/utils';
+
+describe('<Breadcrumbs />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Breadcrumbs />, () => ({
+    classes,
+    inheritComponent: 'nav',
+    render,
+    ThemeProvider,
+    muiName: 'JoyBreadcrumbs',
+    refInstanceof: window.HTMLElement,
+    testVariantProps: { variant: 'solid' },
```

Comment:
```suggestion
    testVariantProps: { variant: 'solid' },
    testCustomVariant: true,
```
If the component accepts `variant` prop, `testCustomVariant: true` should be added to test custom variant.

Reply:
The test is added in #34077